### ISSUE TITLE
Add cli option to request secret app admin/user pin counts

### ISSUE THREAD

**Issue by nestire** [Nitrokey/nitrokey-hotp-verification#39]
see also issue #36 

to prevent complicated parsing of the info output which is not stable create a option that allows requesting the pin counters direct  like _get_secretsapp_adminpin_counter_   _get_secretsapp_userpin_counter_ . which then can be used directly in heads.

**Comment by daringer** [comment]
sounds good, means "admin pin counter" and "user pin counter" will always be identical for the nk3 as this actually is the "secrets pin counter".

**Comment by sosthene-nitrokey** [comment]
For older hardware, If I follow correctly this should return the GPG pin retry counter?

**Comment by nestire** [comment]
correct as it does in the `info` output this way we could simply replace the parsing of the info output with that command. To minimize parsing in heads i would like to just get the number as output, because of this I prefer 2 cli options.
That for NK3 this is the same is fine for backwards compatibility , since the UX will only show the "info" output #38 anyways.

**Comment by tlaurion** [comment]
@JonathonHall-Purism fyi

**Comment by sosthene-nitrokey** [comment]
For the NK3, it is possible that the PIN is not set. The output format will follow:

If the pins are set:

```
Admin PIN Counter: 3
User PIN Counter: 3
```

If the pins are not set:

```
Admin PIN Counter: Not set
User PIN Counter: Not set
```

**Comment by tlaurion** [comment]
@sosthene-nitrokey just want to make sure we are at the same page:

> <nk3
> ```
> HOTP code verification application, version 1.7
> Connected device status:
> Card serial: 0x7BE66C6C
>          Firmware Nitrokey 2: v1.7.1
> OpenPGP smartcard PIN counters : Admin: 3, User: 3
> Operation success
> ```
> 
> nk3:
> ```
> HOTP code verification application, version 1.7
> Connected device status:
> Card serial: 0x7BE66C6C
>          Firmware Nitrokey 3: v1.7.1
>          Firmware Secrets app: v4.13
> Secret app PIN counter : 6
> OpenPGP smartcard PIN counters : Admin: 3, User: 3
> Operation success
> ```

_Originally posted by @tlaurion in https://github.com/Nitrokey/nitrokey-hotp-verification/issues/38#issuecomment-2487450702_

**Comment by sosthene-nitrokey** [comment]
Yes, for the NK3 both PINS are the same, this should make it simpler to parse? I'm implementing as suggested in #36

**Comment by tlaurion** [comment]
I repeat

You proposed :
> See also issue #36 
> Change this:
> ```
> 
> HOTP code verification application, version 1.6
> Connected device status:
> 	Card serial: 0x7BE66C6C
> 	Firmware: v4.13
> 	Card counters: Admin 6, User 6
> Operation success
> ```
> To
> ```
> HOTP code verification application, version 1.6
> Connected device status:
> 	Card serial: 0x7BE66C6C
>         Firmware Nitrokey 3: v1.7.1
> 	Firmware Secrets app: v4.13
> 	Secret app pin counters : Admin 6, User 6
> Operation success
> ```

To my brain, what would make sense for librem key 2 /nk2/ nk1 librem key 1:
```
HOTP code verification application, version 1.7
Connected device status:
Card serial: 0x7BE66C6C
         Firmware Nitrokey 2: v1.7.1
Secret app pin counters : None
OpenPGP smartcard PIN counters : Admin: 3, User: 3
Operation success
```


_Originally posted by @tlaurion in https://github.com/Nitrokey/nitrokey-hotp-verification/issues/38#issuecomment-2487450702_

**Comment by daringer** [comment]
but isn't this the `info` output you are showing an example for?

1) do you want this there (`info`) or here `pin-info`
2) or don't you want two different commands for that ?
3) this issue is about `pin-info` not `info` those are distinct arguments for the tool

**Comment by tlaurion** [comment]
Heads, users, scripts etc simply intelligible output without false info. There is no need to add User/Admin PIN if those are not GPG related for nk3, pin-info is not needed if reported by `info`, correctly, as above.

- no ADMIN/User PIN for nk3 secret app PIN, only secret app PIN exists, only secret App PIN should exist/be presented for nk3 HOTP, where GPG PINs need to be reported if possible (it was agreed this was possible) so that Heads can help the user understand what is going on if HOTP sealing doesn't work.
- As said in seperate issue/PR, secret app pin should always be set, and then not requiring a change pin, simply because there is no admin PIN to unlock secret app pin if all 6 counters have been used: a rest is needed.
  - if secret app PIN is reset with a PIN by default, then physical presence still required twice as of today to set secret app pin and then to change PIN; therefore because of physical presence, it is advise to set a custom PIN in reownership/factory reset so that we don't trigger physical presence twice in the goal of streamlining transfer of ownership.

To this issue:
- there is no need for pin-info, the two functions presented here would be useful internally to provide real and useful info, like GPG PINs for `info` output to provide real state of the security dongles GPG counter info. That is, no secure app PIN for <nk3, secure app PIN for nk3 and GPG user PIN, GPG Admin PIN, useful firmware version and if Nitrokey finds it useful, secret app version.

I keep providing the same output for consistency, Heads uses `hotp_verification info` everywhere, doesn't know about `pin-info` and doesn't need this separately. What we need is true, actionable info for operations and users understanding of what is going on in case of admin PIN failing for <nk3, and secure app PIN for nk3, right? We need the user, Heads to do the right thing in case HOTP sealing doesn't work as intended. For <nk3, this is through gpg admin PIN, for nk3 this is with secure app PIN. The rest will be for documentation purposes, I keep not understanding how secure app user PIN and secure APP admin PIN would be documented as of today: "they are the same" ? Then don't show duplicate and fix `hotp_verification info` to provide "GPG Admin PIN", "GPG User PIN" and "Secure APP PIN" only where/if relevant and true, don't output anything that doesn't exist, don't add new arguments for hotp_verification; just deal with it internally to provided needed output to be actionnable upon by users, heads, and docs.

**Comment by sosthene-nitrokey** [comment]
I think I understand. I have all the devices I need to debug and implement the complete `info` command that includes the device information and the distinct PINs.
